backends/cuda-gen/ceed-cuda-gen-operator.c

```diff
@@ -19,14 +19,21 @@
 #include "ceed-cuda-gen-operator-build.h"
 #include "../cuda/ceed-cuda.h"
 
+//------------------------------------------------------------------------------
+// Destroy operator
+//------------------------------------------------------------------------------
 static int CeedOperatorDestroy_Cuda_gen(CeedOperator op) {
   int ierr;
   CeedOperator_Cuda_gen *impl;
   ierr = CeedOperatorGetData(op, (void *)&impl); CeedChk(ierr);
   ierr = CeedFree(&impl); CeedChk(ierr);
+
```

Comment:
I don't see a purpose to this hanging whitespace.

Reply:
No purpose beyond matching style between functions. Almost all of our functions have it, for some reason.

Reply:
I don't know why. I usually remove it if editing code that has it.

Reply:
We could just start letting it die from this point forward? I don't much care either way; I just try to match style as I notice it.

Reply:
Sounds good.